tasks/main.yml

```diff
@@ -2,7 +2,8 @@
 - import_tasks: auth.yml
 - import_tasks: query.yml
 - import_tasks: config.yml
+- import_tasks: jbpm-console.yml
 - import_tasks: script.yml
 - import_tasks: docker.yml
-- import_tasks: users.yml
+- import_tasks: db.yml
```

Comment:
Maybe `access.yml` would be more descriptive?

Reply:
There is a setting, which has nothing to do with users/access:
```
UPDATE okm_config
SET cfg_value = 'com.openkm.workflow.adapter.JbpmRemoteWorkflowAdapter'
WHERE cfg_key = 'workflow.adapter'
```

So it looks more like config, but inside of DB.